lib/molecule/command/login.py

```diff
@@ -147,19 +142,8 @@ def _get_login(self, hostname):  # pragma: no cover
         login_options["lines"] = lines
         login_cmd = self._config.driver.login_cmd_template.format(**login_options)
 
-        dimensions = (int(lines), int(columns))
         cmd = "/usr/bin/env {}".format(login_cmd)
-        self._pt = pexpect.spawn(cmd, dimensions=dimensions)
```

Comment:
@retr0h Can you please comment on why the pexpect implementation was added in the first place for the interactive login? I am asking because I would not want to introduce a serious regression just because I do not know the reasons behind original implementation.

I did some basic tests and using subprocess.run() appeared to be working fine for me.

Reply:
Possible reason from the docs: "The run() function was added in Python 3.5"

If we previously supported < 3.5, then this call wouldn't have been possible.